packages/storage-adapter-spec/src/index.js

```diff
@@ -66,6 +88,11 @@ export default function runAbstractStorageAdapterSpec(adapter) {
           .then(() => assert.becomes(bound.get(key), obj))
           .then(() => assert.isFulfilled(bound.del(key)))
           .then(() => assert.isRejected(bound.get(key))));
+
+        it(`removes an item from the store when putting \`undefined\``, () =>
+          bound.put(key, undefined)
```

Comment:
When relying on implicit return, it's best to put the first line on the same line as the arrow so it's clear the braces were omitted intentionally.